core/block/editor/state/normalize.go

```diff
@@ -292,6 +321,49 @@ func (s *State) normalizeSmartBlock(b simple.Block) {
 	}
 }
 
+func (s *State) shortenDetailsToLimit(details map[string]*types.Value) {
+	for key, value := range details {
+		size := value.Size()
+		if size > detailSizeLimit {
+			log.With("objectID", s.rootId).Errorf("size of '%s' detail (%d) is above the limit of %d. Shortening it",
+				key, size, detailSizeLimit)
+			value, _ = shortenValueOnN(value, size-detailSizeLimit)
+		}
+	}
+}
+
+func shortenValueOnN(value *types.Value, n int) (result *types.Value, left int) {
```

Comment:
Does this function try to remove N bytes from value?

Reply:
Yes

Reply:
shortenOn sounds like we short value **on** specific position. Maybe use **by** preposition?